include/oneapi/dpl/ranges

```diff
@@ -25,7 +27,7 @@
 #    define _ONEDPL_CXX_STANDARD_LIBRARY_CHECK 1
 #endif
 
-static_assert(__cplusplus >= 201703L && _ONEDPL_CXX_STANDARD_LIBRARY_CHECK,
+static_assert(_ONEDPL_cplusplus >= 201703L && _ONEDPL_CXX_STANDARD_LIBRARY_CHECK,
               "The use of the range-based API requires C++17 and the C++ standard libraries coming "
               "with GCC 8.1 (or higher) or Clang 7 (or higher)");
```

Comment:
Should we only leave the second part of the check and update the comment, as now C++17 is always a requirement? @MikeDvorskiy what do you think?

Reply:
> Should we only leave the second part of the check and update the comment, as now C++17 is always a requirement? @MikeDvorskiy what do you think?

yes, it makes sense to leave just the second part of the comment.